LICENSE

```diff
@@ -175,3 +175,8 @@
       of your accepting any such warranty or additional liability.
 
    END OF TERMS AND CONDITIONS
+
+
+This product contains code from Rackspace GopherCloud, which is available
+under an Apache License 2.0. For details, see openstack/v2/snapshots/requests.go, openstack/v2/snapshots/results.go,
+openstack/v2/volumes/requests.go, openstack/v2/volumes/results.go
```

Comment:
In mentioned files (openstack/v2/snapshots/requests.go, openstack/v2/snapshots/results.go, openstack/v2/volumes/requests.go, openstack/v2/volumes/results.go) if there are original to Intel but with snippets from 3rd party components - files should have two Copyrights headers: Intel + 3rd component, see [this](https://github.com/intelsdi-x/snap-plugin-publisher-heapster/blob/master/exchange/exchange.go) example.